pkg/diag/colors/colors.go

```diff
@@ -15,30 +15,25 @@
 package colors
 
 import (
+	"bytes"
 	"fmt"
 	"strings"
 
-	"github.com/reconquest/loreley"
-
 	"github.com/pulumi/pulumi/pkg/util/contract"
 )
 
 const colorLeft = "<{%"
 const colorRight = "%}>"
 
-func init() {
-	// Change the Loreley delimiters from { and }, to something more complex, to avoid accidental collisions.
-	loreley.DelimLeft = colorLeft
-	loreley.DelimRight = colorRight
-}
+var disableColorization bool
 
-func Command(s string) string {
+func command(s string) string {
 	return colorLeft + s + colorRight
 }
 
-// TrimPartialCommand returns the input string with any partial colorization command trimmed off of the right end of
+// TrimPartialcommand returns the input string with any partial colorization command trimmed off of the right end of
 // the string.
-func TrimPartialCommand(s string) string {
+func TrimPartialcommand(s string) string {
```

Comment:
Nit - but this name change doesn’t feel necessary.

Reply:
Oh, that was unintentional. Bad search-replace.